### PR TITLE
boa: add clean script

### PR DESCRIPTION
**Pull request by FeelyChau**
This PR implemented `clean` script for the `boa` package.

### CODE REVIEW

File: packages/boa/tools/clean-python.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const { run, CONDA_DOWNLOAD_NAME, CONDA_INSTALL_DIR } = require('./utils');
+const fs = require('fs');
+
+let CONDA_DOWNLOAD_PREFIX = 'https://repo.anaconda.com/miniconda';
+if (process.env.BOA_CONDA_MIRROR) {
+  CONDA_DOWNLOAD_PREFIX = process.env.BOA_CONDA_MIRROR;
```
Comment (yorkie) [inline packages/boa/tools/clean-python.js:8]: This doesn't get used :)

File: packages/boa/tools/clean-python.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+if (fs.existsSync(CONDA_DOWNLOAD_NAME)) {
+  run('rm', `-f ./${CONDA_DOWNLOAD_NAME}`);
+}
+  
+if (fs.existsSync(CONDA_INSTALL_DIR)) {
+  run('rm', `-rf \`cat ${CONDA_INSTALL_DIR}\``);
+  run('rm', `-f ${CONDA_INSTALL_DIR}`);
+}
```
Comment (yorkie) [inline packages/boa/tools/clean-python.js:18]: Maybe bash is better for this script?

**Comment by yorkie** [comment]
And there are the following instructions:

1. sign the Alibaba CLA for contribution.
2. update your git user/email for your GitHub account.

File: packages/boa/tools/clean-python.js (hunk 1 above)
Comment (FeelyChau) [reply packages/boa/tools/clean-python.js]: Done.

File: packages/boa/tools/clean-python.js (hunk 2 above)
Comment (FeelyChau) [reply packages/boa/tools/clean-python.js:13]: Do you mean to put the script in a bash file?

Comment (yorkie) [reply packages/boa/tools/clean-python.js:13]: I mean rewrite in Bash

**Comment by yorkie** [comment]
> May I ask you to add the clean script only?

And you need to remove changes of others.

File: packages/boa/tools/clean-python.js (hunk 1 above)
Comment (FeelyChau) [reply packages/boa/tools/clean-python.js]: Done.

> > May I ask you to add the clean script only?
> 
> And you need to remove changes of others.

Done.

File: packages/boa/tools/clean-python.js (hunk 2 above)
Comment (FeelyChau) [reply packages/boa/tools/clean-python.js]: OK.

**Pull request merged**